android/settings.gradle

```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'atb'
+include ':react-native-localize'
```

Comment:
This shouldn't be necessary since packages will autolink 0.60+

Reply:
Ah, wasn't aware of that. :thinking: